File: IntegraAppFront/src/app/models/Empleado/empleado.model.ts
export class Empleado {
  id: number | null;
  apellido: string | null;
  nombre: string | null;
  telefono: string | null;
  correo: string | null;
  fechaContratacion: Date;
  archivo: File | null;

  constructor(
    id: number | null,
    apellido: string | null,
    nombre: string | null,
    telefono: string | null,
    correo: string | null,
    fechaContratacion: Date,
    archivo: File | null
  ) {
    this.id = id;
    this.apellido = apellido;
    this.nombre = nombre;
    this.telefono = telefono;
    this.correo = correo;
    this.fechaContratacion = fechaContratacion;
    this.archivo = archivo;
  }
}

File: src/IntegraApp.Domain/Models/EmpleadoModel.cs
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;

namespace IntegraApp.Domain.Models
{
    [Table("Empleados")]
    public class EmpleadoModel
    {
        [Key]
        public int Id { get; set; } 
        public string? Apellido { get; set; }
        public string? Nombre { get; set; }
        public string? Telefono { get; set; }
        public string? Correo { get; set; }
        public string? FotoPath { get; set; } 
        public DateTime FechaContratacion { get; set; }
    }
}
